server/@types/forms/index.d.ts

```diff
@@ -9,11 +9,13 @@ declare module 'forms' {
   }
 
   export interface AddStepForm {
+    stepNumber: number
     title?: string
     targetDate?: Date
     'targetDate-day'?: string
     'targetDate-month'?: string
     'targetDate-year'?: string
+    action?: string
```

Comment:
I'm not sure about the name `action` (it's a bit vague), but I can't think of a better one right now! 🤔

Reply:
I did consider `formAction`, but decided against it because a) its in a form type anyway so felt that was already implied; and b) "form action" means something subtly different in context of a HTML form (it has an `action` attribute which tells the browser where to post the data to)